Main.java

```diff
@@ -9,7 +11,22 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //Enter line that asks the user to enter an interger
+    System.out.println("Please enter an interger:");
+    int a = input.nextInt();
+
+    //Enter line that asks the user to enter another interger
+    System.out.println("Please enter another interger:");
+    int b = input.nextInt();
```

Comment:
a and b would generally be considered bad variable names. Remember, variables should always be named using camelCase and describe what they are storing or being used for.